docs/helm/desktop-deployment.md

```diff
@@ -1,57 +1,53 @@
-- Latest version of [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) is installed
-- Latest version of [Helm](https://helm.sh/docs/intro/install) is installed
+- [Rancher for Desktop](https://rancherdesktop.io/). Rancher Desktop includes kubectl and Helm as pre-installed tools, ready to use right after installation.
 
-The following instructions are also applicable to [Rancher Desktop](https://rancherdesktop.io/) users. Rancher desktop will provide `kubectl` and `helm` out of the box. it is open source but less mature than Docker desktop.
-
-## Configure Docker for Desktop
-
-In order to deploy onto Docker for Desktop we need to configure it with as much CPU and memory as possible on the "Resources" tab in Docker for Desktop's preferences pane as shown in the screenshot below. Similar resource requirement apply to Rancher Desktop.
+### Rancher Desktop specific configuration
 
-![Resources](./diagrams/dfd-resources.png)
+Uncheck `Enable Traefik` from the `Kubernetes Settings` page to disable Traefik. You may need to exit and restart Rancher Desktop for the change to take effect. Ref: [Setup NGINX Ingress Controller](https://docs.rancherdesktop.io/how-to-guides/setup-NGINX-Ingress-Controller)
 
-### Docker for Desktop specific configuration
+### Docker Desktop specific configuration
 
-To deploy the Helm charts Kubernetes needs to be enabled, this can be done from the "Kubernetes" tab, as shown in the screenshot below. Press the "Apply & Restart" button to confirm.
+[Prerequisites](./desktop-deployment.md#Prerequisites) are consistent with those for Docker Desktop. Additionally, it is essential to install the latest version of [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) & [Helm](https://helm.sh/docs/intro/install).
 
-![k8s Enabled](./diagrams/dfd-k8s-enabled.png)
+After the installation of Docker Desktop, the following configurations should be adjusted within Docker Desktop settings.
+`Settings > Resources > Advanced > CPUs:8, Memory: 16GB, Swap: 1GB`
+`Settings > kubernetes > Enable Kubernetes`
 
-### Rancher Desktop specific configuration
+After making the necessary settings `Apply and restart` the docker desktop.
 
-Uncheck `Enable Traefik` from the `Kubernetes Settings` page to disable Traefik. You may need to exit and restart Rancher Desktop for the change to take effect. Ref: [Setup NGINX Ingress Controller](https://docs.rancherdesktop.io/how-to-guides/setup-NGINX-Ingress-Controller)
+## Deployment
 
-## Deploy
+After successfully launching Rancher Desktop, proceed with the instructions provided in the following sections to install ACS (Enterprise or Community) on your local system.
 
-Once Docker for Desktop is running follow the steps in the following sections to deploy ACS (Enterprise or Community) to your local machine.
+> Note: To execute the deployment on Docker Desktop, please adhere to the same set of deployment instructions provided below.
```

Comment:
do not reference Rancher or Docker desktop from here onwards, use a generic reference like local machine or similar